specs/swagger.yaml

```diff
@@ -341,14 +351,76 @@ paths:
             $ref: "#/definitions/Peptide"
         '400':
           description: Bad request. The requested parameters are not supported.
-        '401':
-          description: Authorization information is missing or invalid.
+          schema:
+            $ref: "#/definitions/Error"
+        '404':
+          description: No data found for the specified parameters.
+          schema:
+            $ref: "#/definitions/Error"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+  /proteins:
+    get:
+      summary: Get a collection of proteins
+      operationId: getProteins
+      description: The protein entreypoint returns a protein statistics across an entire resource.
```

Comment:
1. Misspelled "entry point".
2. Remove "a" since "protein statistics" is plural.